.github/workflows/gapjl.yml

```diff
@@ -40,10 +41,18 @@ jobs:
 
     steps:
       - name: Checkout GAP.jl
+        run: |
+          using Pkg
+          if ${{ matrix.gapjl-version }} == "latest-release"
+            Pkg.add("GAP")
+          else
+            Pkg.add(;name="GAP", rev="${{ matrix.gapjl-version }}")
+          end
+          path = dirname(dirname(Base.find_package("GAP")))
+          open(ENV["GITHUB_ENV"], "a") do io
+              println(io, "GAPJLPATH=$path")
+          end
         uses: actions/checkout@v4
```

Comment:
```suggestion
```
I think this was a syntax error

Reply:
Yeah, github needed a minute longer than expected to show it in https://github.com/gap-system/gap/actions/runs/9370508249